src/lib/jsonParseV2/parseContextAttribute.cpp

```diff
@@ -121,11 +121,54 @@ static std::string parseContextAttributeObject(const Value& start, ContextAttrib
         return r;
       }
     }
-    else  // ERROR
+    else if (keyValues == false) // ERROR
     {
       LM_W(("Bad Input (unrecognized property for ContextAttribute - '%s')", name.c_str()));
       return "unrecognized property for context attribute";
     }
+    else  // Anything captured here is a metadata, and options=keyValues has been set
```

Comment:
We are inside an attribute here (and keyValues is *true*), so if a field named "m1" is encountered, then it is not an error, but a metadata (without keyValues true) it would be an error.